packages/mobile/src/screens/audio-screen/ChallengeRewards.tsx

```diff
@@ -145,7 +219,47 @@ export const ChallengeRewards = () => {
       {userChallengesLoading && !haveChallengesLoaded ? (
         <LoadingSpinner style={styles.loading} />
       ) : (
-        rewardsPanels
+        <Flex gap='2xl'>
+          {isRewardsCooldownEnabled ? (
+            <Paper shadow='flat' border='strong' p='l' gap='m'>
+              <Flex direction='row' justifyContent='flex-start' gap='s'>
+                {claimableAmount > 0 ? (
+                  <IconTokenGold height={24} width={24} />
+                ) : null}
+                <Text variant='heading' color='accent' size='s'>
+                  {messages.totalReadyToClaim}
+                </Text>
+              </Flex>
+              <View style={styles.pillContainer}>
```

Comment:
is there an off-the-shelf pill component we can use? lets chat about a plan here.

Reply:
i think we talked about this before when doing web and decided to just copy + paste the existing one to make it consistent but lmk if we should do something else

https://github.com/AudiusProject/audius-protocol/blob/79d501cf6f0403860cbae717fde7ad4ba474e3a9/packages/web/src/pages/audio-rewards-page/ChallengeRewardsTile.tsx#L287-L292